### PR TITLE
Fix parse errors when property pattern is explicit.

### PR DESCRIPTION
**Pull request by mgonzalezperna**
Proposed workaround to fix parse error when declaring regex. (https://github.com/koxudaxi/datamodel-code-generator/issues/124)

### CODE REVIEW

File: datamodel_code_generator/model/pydantic/types.py [hunk 1]
```diff
@@ -134,6 +134,8 @@ def get_data_float_type(types: Types, **kwargs: Any) -> DataType:
 def get_data_str_type(types: Types, **kwargs: Any) -> DataType:
     data_type_kwargs = transform_kwargs(kwargs, string_kwargs)
     if data_type_kwargs:
+        if 'regex' in data_type_kwargs.keys():
```
Comment (koxudaxi) [inline datamodel_code_generator/model/pydantic/types.py]: I think `.keys()` is not need.

**Comment by koxudaxi** [comment]
@mgonzalezperna 
Thank you for creating This PR.

I wrote a comment, Otherwise, it's OK :smile:

File: datamodel_code_generator/model/pydantic/types.py (hunk 1 above)
Comment (mgonzalezperna) [reply datamodel_code_generator/model/pydantic/types.py]: Fixed!

**Comment by koxudaxi** [comment]
@mgonzalezperna 
I have merged and published a new version as `0.4.10` :tada: 

Thank you very much :+1:

**Pull request merged**